src/pages/api-showcase.tsx

```
@@ -0,0 +1,459 @@
+import { useSearchState } from '@/hooks/useUrlState';
+import { withServerSideConfig } from '@/lib/config/next';
+import { importedConfigurations } from '@/config/configs';
+
+interface DataSection {
+  id: string;
+  title: string;
+  description: string;
+  loader: (
+    good: SafeApiClient<AlcoholZincApi<unknown>>,
+    bad: SafeApiClient<AlcoholZincApi<unknown>>,
+  ) => Promise<Result<any, Problem>>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)